index.json

```diff
@@ -1,7 +1,8 @@
 {
   "name": "client-widget-CREA-fcpo-specials-banner",
   "beta": true,
-
+  "is_layout": true,
```

Comment:
The CMS no longer hard-codes a list of widget names that are "layout". It now relies on "is_layout"